src/fork.rs

```diff
@@ -331,6 +331,7 @@ const SUPPORTED_VERSIONS: &[ProtocolVersionId] = &[
     ProtocolVersionId::Version17,
     ProtocolVersionId::Version18,
     ProtocolVersionId::Version19,
+    ProtocolVersionId::Version20,
```

Comment:
Is this the latest version? I thought I saw @mm-zk getting an error for `Version21`